README.md

```diff
@@ -98,3 +107,9 @@ Access the playwright code generator tool with:
 ```bash
 pnpm run test:codegen
 ```
+
+## Troubleshooting
+### `address already in use` (Port 5000) on macOS
+macOS AirPlay Receiver uses port 5000 by default, which conflicts with Nigiri.
```

Comment:
Not entirely convinced about this: it's a nigiri issue on MacOS. Maybe it belongs to that README ?